concourse/scripts/compile_gpdb.bash

```diff
@@ -151,7 +151,7 @@ function _main() {
 		unittest_check_gpdb
 	fi
 
-	#include_dependencies
+#	include_dependencies
```

Comment:
Is this a necessary change?

Reply:
Yes. We do not support vendor library builds. This line is also commented out in 7X.

Reply:
I'm talking about the change in the position of the hash symbol.

Reply:
fixed